mir_eval/separation.py

```diff
@@ -52,9 +52,9 @@ def validate(reference_sources, estimated_sources):
 
     Parameters
     ----------
-    reference_sources : np.ndarray, shape=(nsrc, nsampl)
+    reference_sources : np.ndarray, shape=(nsrc1, nsampl1)
```

Comment:
Since these are required to be the same shape (as per the `ValueError` when `reference_sources.shape != estimated_sources.shape`), why change the docstring like this?